Makefile

```diff
@@ -78,6 +79,15 @@ $(warning No code-sign identity found (CODE_SIGN_IDENTITY is not set))
 endif
 endif
 
+ifeq ($(TARGET_PLATFORM),win32)
+ifndef CODE_SIGN_CERTIFICATE
+$(warning No code-sign certificate found (CODE_SIGN_CERTIFICATE is not set))
+endif
+ifndef CODE_SIGN_CERTIFICATE_PASSWORD
+$(warning No code-sign certificate password found (CODE_SIGN_CERTIFICATE_PASSWORD is not set))
+endif
```

Comment:
I guess one option would be to have a non-signed version as the "standard" DMG, and then have a special code-signed DMG required by the publish step? That's probably a bit too much of a step though just to silence a couple of warnings ;-)
Or a simple solution could just be a `SILENCE_CODE_SIGN_WARNINGS ?= 0` in the Makefile, so the user could set a `SILENCE_CODE_SIGN_WARNINGS=1` env-var if they wanted to.

**OTOH**, perhaps it's inconsistent behaviour to have the Makefile generate two different versions of the same file, based on the value of an env-var? (makes it hard to tell them apart?)
Consider if:
 1. You run make without the CODE_SIGN env-vars set
 2. Make creates the unsigned package
 3. You run make again _with_ the CODE_SIGN env-vars set
 4. Make doesn't do anything, because it thinks the package is already up to date
 5. You run 'make publish'
 6. Whoops, make has now uploaded the unsigned packages!

What if instead, there was e.g. a `release/out/signed` directory, and `make publish` depended on `release/out/signed/Etcher-1.0.0-beta.17-darwin-x64.dmg`, which in turn depended on the temporary read-write DMG (could it be given a different name and made non-temporary?). And then `release/out/Etcher-1.0.0-beta.17-darwin-x64.dmg` would also depend on the same 'temporary' read-write DMG, and would always be the non-signed package.
Would that work? ;)